sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/clustering/algorithms/SilviaClustering.scala

```diff
@@ -432,6 +434,10 @@ object SilviaClustering {
         unionList = List()
 
       }
+      // spark intialization
+      val spark = SparkSession.builder.getOrCreate()
+       import spark.implicits._
+
       val rdfRDD = spark.sparkContext.parallelize(rdfString)
```

Comment:
What is this `rdfString`? Does it contain a lot of data? How big it can be?

Reply:
it was already there, I just changed some implementation- earlier sparkSession was sent as the argument to the class, now creating the sparkSession context in the code itself.